### PR TITLE
New version

### PR DESCRIPTION
**Pull request by dennis-montana**
Created new plugin for Salesforce DMP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,54 @@
...
 
-## Usage 
+## Usage
 
-Describe any usage specifics for your plugin. Give examples for Android, iOS, Angular if needed. See [nativescript-drop-down](https://www.npmjs.com/package/nativescript-drop-down) for example.
-	
-	```javascript
-    Usage code snippets here
-    ```)
+To use nativescript-ng-sentry you must first import the module:
```
Comment (bartsg) [inline README.md]: Wrong plugin name?

File: README.md [hunk 2]
```diff
@@ -1,40 +1,54 @@
...
+```ts
+import { SalesForce, KeyValue } from '@essent/nativescript-salesforce';
+```
 
-## API
+At the launch of your app call `initialize` with your config id:
+```ts
+SalesForce.getInstance().initialize('abcdefghij', true);
```
Comment (bartsg) [inline README.md]: replace abcdefghij with YOUR_CONFIG_ID or something similar

File: demo/app/main-page.ts [hunk 3]
```diff
@@ -1,10 +1,27 @@
...
-import {HelloWorldModel} from './main-view-model';
+import { SalesForce, KeyValue } from '@essent/nativescript-salesforce';
 
-// Event handler for Page 'loaded' event attached in main-page.xml
-export function pageLoaded(args: observable.EventData) {
-    // Get the event sender
-    let page = <pages.Page>args.object;
-    page.bindingContext = new HelloWorldModel();
+export function pageLoaded(_args: observable.EventData) {
+    SalesForce.getInstance().initialize('abcdefghij', true);
```
Comment (bartsg) [inline demo/app/main-page.ts]: Same here, would be more readable when replaced by YOUR_CONFIG_ID or so.

**Review by bartsg — APPROVED** [review]

**Comment by corne-de-bruin** [comment]
First of all i would like to propose to rename this plugin to nativescript-salesforce-dmp because that really what's supported. Were not supporting the generic salesforce business

**Review by corne-de-bruin — CHANGES_REQUESTED** [review]
Again I would like to propose to rename all salesforce reference to salesforce-dmp

**Review by martijnkooijman — APPROVED** [review]

**Pull request merged**